cellprofiler/modules/measureimageareaoccupied.py

```diff
@@ -133,21 +134,21 @@ def __init__(self):
                 self.__image_name = cellprofiler.setting.ImageNameProvider(
                     "Name the output binary image",
                     "Stain",
-                    doc="""
-                    <i>(Used only if the binary image of the objects is to be retained for later use in the
-                    pipeline)</i><br>
-                    Specify a name that will allow the binary image of the objects to be selected later in the
-                    pipeline.
+                    doc="""\
+*(Used only if the binary image of the objects is to be retained for
```

Comment:
Add a newline after *(Used only if...)* text to insert a line break in the documentation rendered in the GUI.